tests/CMakeLists.txt

```diff
@@ -214,3 +214,23 @@ add_test(
 set_tests_properties("test-create_scap_delta_tailoring_resolved" PROPERTIES FIXTURES_REQUIRED "rule-dir-json")
 set_tests_properties("test-create_scap_delta_tailoring_resolved" PROPERTIES DEPENDS "test-rule-dir-json")
 endif()
+
+if (PYTHON3_EXECUTABLE AND PYTHON_VERSION_MINOR GREATER 6)
+    add_test(
+        NAME "test-controleval-directory"
+        COMMAND env "PYTHONPATH=$ENV{PYTHONPATH}" "${PYTHON_EXECUTABLE}" "${CMAKE_SOURCE_DIR}/utils/controleval.py" "--controls-dir" "${CMAKE_SOURCE_DIR}/controls" "stats" "--level" "medium" "--product" "rhel9" "--id" "srg_gpos"
+    )
+    set_tests_properties("test-controleval-directory" PROPERTIES LABELS quick)
+
+    add_test(
+        NAME "test-controleval-onefile"
+        COMMAND env "PYTHONPATH=$ENV{PYTHONPATH}" "${PYTHON_EXECUTABLE}" "${CMAKE_SOURCE_DIR}/utils/controleval.py" "--controls-dir" "${CMAKE_SOURCE_DIR}/controls" "stats" "--level" "l2_server" "--product" "rhel8" "--id" "cis_rhel8"
```

Comment:
Are we sure that the RHEL 8 CIS will be always implemented in a single file? If somebody in future decides to split it into multiple control files, then the test will become similar to the test above.

Reply:
There is no guarantee. But it would take a decent amount of work to split it up. We could use some of the test controls files that the control file unit tests use or even create our own for this test.

Reply:
Yes, it would be better to use a small controls file which would be located in the tests directory

Reply:
I have moved that check to use a test file. So we want to move the others. If so, we will need to create new ones as the existing test ones don't have levels.